src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -27,16 +27,16 @@
   "minecolonies.config.diseases.comment": "All diseases citizens can get. The name, how common it is, and all ingredients to cure it.",
 
   "minecolonies.config.generatesupplyloot": "Supply loot",
-  "minecolonies.config.generatesupplyloot.comment": "Whether to generate supply ship/camps in chests occasionally [Default: true]",
+  "minecolonies.config.generatesupplyloot.comment": "Whether to generate supply ship/camps in chests occasionally. [Default: true]",
 
   "minecolonies.config.listofplantables": "Florist Flowers",
-  "minecolonies.config.listofplantables.comment": "The items and item-tags that the florist can plant.",
+  "minecolonies.config.listofplantables.comment": "The items and item-tags that the Florist can plant.",
 
-  "item.minecolonies.scepterlumberjack.name": "Lumberjack-scepter",
-  "item.minecolonies.scepterlumberjack.usedstart": "First, position set.",
-  "item.minecolonies.scepterlumberjack.usedend": "Second, position set.",
-  "item.minecolonies.scepterlumberjack.restrictiontoobig": "Area, %s is too large. Maximum restricted area for the lumberjack is %s blocks.",
-  "item.minecolonies.scepterlumberjack.restrictionset": "Lumberjack, restricted area set. Size is %s blocks. Max size is %s blocks.",
+  "item.minecolonies.scepterlumberjack.name": "Logger-scepter",
```

Comment:
Lumberjack to Logger? That seems like a weird name, no?

Reply:
It is a direct synonym, and is definitely called that in some places.
Lumberer is also a valid term, apparently. Maybe this is closer to what you are expecting?